### PR TITLE
Branding sciencebeam

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

File: demo/package-lock.json
Comment (de-code) [inline demo/package-lock.json:1717]: just adding that there seem to be automatic changes adding `optional` in many places but no dependency seem to have changed.

**Comment by de-code** [comment]
Thank you for doing that.

The loading message seems to be difficult to read (probably you didn't spot it because of the Docker issues you had):

image

EDIT: Chris had suggested to remove the positive loading message altogether and the invsion show the error message underneath (which you could get by uploading a file in the wrong format for example).

File: demo/src/index.html
Comment (de-code) [inline demo/src/index.html:96]: The model selection seem to work. Although the `value` doesn't seem to change. Not sure if it matters though.

EDIT: never mind. The checked state changes. I didn't have my coffee yet.

**Comment by discodavey** [comment]
The value that comes out is now true or false. If you look in app.js you will see the changes

File: demo/src/js/App.js
Comment (de-code) [inline demo/src/js/App.js]:
```suggestion
```

File: demo/src/js/App.js
Comment (de-code) [inline demo/src/js/App.js]: probably remove commented out code

File: package-lock.json
Comment (de-code) [inline package-lock.json]: I don't think we need this file at the root. There is one in the `demo` folder.

**Comment by de-code** [comment]
Just want to check that the styling change in Texture was intended or desired (it looks better to me):

before:

image

after:

image

**Comment by discodavey** [comment]
That is a question for Chris H but I cannot seem to add him on this PR. Will check with him by commenting on the main ticket.

**Comment by de-code** [comment]
> That is a question for Chris H but I cannot seem to add him on this PR. Will check with him by commenting on the main ticket.

I added Chris to the repo and as a reviewer.

**Comment by de-code** [comment]
This looks all good to me. Just need to address the loading / error message I think: https://github.com/elifesciences/sciencebeam-texture/pull/50#issuecomment-646495104

File: demo/src/js/App.js
Comment (NuclearRedeye) [inline demo/src/js/App.js]: I'd say you can get rid of console.logs() now.

File: demo/src/js/App.js
Comment (NuclearRedeye) [inline demo/src/js/App.js]: I'd say you can get rid of console.logs() now.

File: demo/src/index.html
Comment (NuclearRedeye) [inline demo/src/index.html:96]: Might be able to delete this actually as not used, but not that important to be honest.

**Review by NuclearRedeye — COMMENTED** [review]
Note that I've not tested this locally, might try later. Some minor comments, up to you if you want to address them or not.

**Comment by discodavey** [comment]
> This looks all good to me. Just need to address the loading / error message I think: [#50 (comment)](https://github.com/elifesciences/sciencebeam-texture/pull/50#issuecomment-646495104)

Are you going to deal with this loading issue @de-code ?

**Comment by de-code** [comment]
> > This looks all good to me. Just need to address the loading / error message I think: [#50 (comment)](https://github.com/elifesciences/sciencebeam-texture/pull/50#issuecomment-646495104)
> 
> Are you going to deal with this loading issue @de-code ?

I was referring to a styling issue of the loading / error feedback. Which I thought you would cover?

**Comment by discodavey** [comment]
Sorry.. I thought you mean't the error we were getting. I will deal with the styling. @chugginselifesciences What colours do you want boss?

**Comment by discodavey** [comment]
@de-code I cannot seem to get the local docker version of the site working so it accepts files. I keep getting a network error. Can you have a look as to why please?

**Comment by chris-huggins** [comment]
@discodavey Could you please use `#CF0C4E` for the red error text shown below. I recommended to hide the green text shown when the file is uploaded so don't think there are any others.

@de-code I'd also recommend changing the error text to be a bit more helpful to understand what went wrong if possible.

image

Thanks

**Comment by discodavey** [comment]
@chugginselifesciences I have got to this....

Screenshot 2020-06-23 at 08 59 44

The current green for success is our green background
https://ui-patterns.elifesciences.org/patterns/00-atoms-components-info-bar-success/00-atoms-components-info-bar-success.html

which is not accessibile if the green is used for text. Can you give a suggestion on what colour we can use.

**Comment by chris-huggins** [comment]
@discodavey I recommended to Daniel that we just do not show this success feedback. The action of uploading a file should result in either a spinner (while converting), or an error state. The success state just confuses things. So I would suggest hiding it if you can instead.

**Comment by discodavey** [comment]
@de-code Can you please remove the success message please from the demo as per @chugginselifesciences comment above. 

https://github.com/elifesciences/sciencebeam-texture/pull/50#issuecomment-648004758

@chugginselifesciences Once the changes are merged then can you review the layout etc with your feedback.

**Review by chris-huggins — APPROVED** [review]

**Comment by de-code** [comment]
> @de-code Can you please remove the success message please from the demo as per @chugginselifesciences comment above.

Hi @discodavey would you mind doing that? You can just remove `this.messageBoard.announceSuccess` from `FileHandler.js`.

**Comment by discodavey** [comment]
@de-code I have removed the line for Success. Can you please review and approve if happy.

**Comment by de-code** [comment]
Hi @discodavey the spinner seems to be on the right hand side. Certainly no worse than before but Chris might prefer it to be different:

image

I removed the other call to `announceSuccess` (`File data processed, stored in local storage`).

**Comment by discodavey** [comment]
@de-code Are you happy for me tto squash and merge? Will this push the code to the url so @chugginselifesciences can review the page layout?

**Comment by de-code** [comment]
> @de-code Are you happy for me tto squash and merge?

Okay, up to you. I thought you might want to update the spinner location but could do that later.

> Will this push the code to the url so @chugginselifesciences can review the page layout?

It will currently require a GitHub release which I can quickly do (see https://github.com/elifesciences/issues/issues/5699#issuecomment-646509452).

**Review by de-code — APPROVED** [review]

**Comment by discodavey** [comment]
@de-code Are you ok for me to merge so you can do your bit with GitHub?

**Comment by de-code** [comment]
> @de-code Are you ok for me to merge so you can do your bit with GitHub?

Go for it

([as mentioned above](https://github.com/elifesciences/sciencebeam-texture/pull/50#issuecomment-648126805), the spinner could probably do with slightly more TLC but could do that later)

**Comment by discodavey** [comment]
@de-code Has now been merged

**Comment by de-code** [comment]
It's now live

**Pull request merged**
